typescript/mint-specific-nft/pages/api/getNfts.ts

```diff
@@ -0,0 +1,95 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import fs from 'fs';
+import { ThirdwebSDK } from '@thirdweb-dev/sdk';
+import { ethers } from 'ethers';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export interface NFT {
```

Comment:
you can get this interface from our SDK, feel free to extend it if you need "minted" and "price", its called `NFTMetadataInput`, create an interface and share it between the frontend and the backend